sdk/communication/ci.yml

```diff
@@ -26,6 +26,8 @@ extends:
     ServiceDirectory: communication
     ArtifactName: packages
     Artifacts:
+    - name: Azure.Communication.ShortCodes
```

Comment:
Adding this here might may lead to accidentally publishing the package to the public nuget feed. Unless there is not internal pipeline created for this

Reply:
@chidozieononiwu But isn't it also necessary for releasing as private preview?

Reply:
I see. I just removed it, thanks!